dev/com.ibm.ws.security.oidc.client_fat.spnego/bnd.bnd

```diff
@@ -30,4 +30,10 @@ fat.project: true
     com.ibm.ws.security.spnego_fat;version=latest,\
     com.ibm.ws.security.spnego.fat.common;version=latest,\
     com.ibm.ws.security.token.s4u2;version=latest,\
-    net.sourceforge.htmlunit:htmlunit;version=2.44.0
+    org.apache.httpcomponents:httpclient;version=4.1.2,\
+    org.apache.httpcomponents:httpcore;version=4.1.2,\
+    net.sourceforge.htmlunit:htmlunit;version=2.44.0,\
+    net.sourceforge.htmlunit:htmlunit-cssparser;version=1.6.0,\
+	net.sourceforge.htmlunit:webdriver;version=2.6,\
+    xalan:xalan;version=2.7.2, \
+    org.brotli:dec;version=0.1.2
```

Comment:
If these weren't needed to _build_ the FAT before, I don't think they'd be needed now. I could be mistaken, but I would think the dependencies need to be included in the autoFVT folder to fix this particular problem, which is what your build.gradle changes should be accomplishing.